script.js

```diff
@@ -1,136 +1,451 @@
+    div.className = item;
+    if (div.className === 'hdg') {
+      const hdg = div;
+      popUpInfo.map((elem) => {
+        elem.classes.map((item) => {
+          if (item === 'hdr') {
+            const div = document.createElement('div');
+            const hdr = div;
+            div.className = item;
+            popUpInfo.map((item) => {
+              item.heading.map((e) => {
+                if (e === 'Multi Post Stories') {
+                  const h2 = document.createElement('h2');
+                  h2.innerHTML = e;
+                  popUpInfo.map((elem) => {
+                    elem.classes.map((item) => {
+                      if (item === 'heading1') {
+                        h2.className = item;
+                        hdr.appendChild(h2);
+                        hdg.appendChild(hdr);
+                      }
+                    });
+                  });
+                }
+                if (e === 'Keeping track of hundreds of components website') {
+                  const h2 = document.createElement('h2');
+                  h2.innerHTML = e;
+                  popUpInfo.map((elem) => {
+                    elem.classes.map((item) => {
+                      if (item === 'heading2') {
+                        h2.className = item;
+                        hdr.appendChild(h2);
+                        hdg.appendChild(hdr);
+                      }
+                    });
+                  });
+                }
+              });
+              item.classes.map((e) => {
+                if (e === 'fa-solid fa-x') {
+                  const span = document.createElement('span');
+                  const icn = document.createElement('i');
+                  const popIcon = icn;
+                  popIcon.addEventListener('click', () => {
+                    blur.classList.remove('active');
+                    openPopUp.classList.remove('active');
+                  });
+                  icn.className = e;
+                  span.appendChild(icn);
+                  hdr.appendChild(span);
+                }
+              });
+              const ul = document.createElement('ul');
+              item.technologies.map((e) => {
+                const li = document.createElement('li');
+                li.innerHTML = e;
+                ul.appendChild(li);
+                hdg.appendChild(ul);
+              });
+            });
+          }
+        });
+      });
+    }
+    if (div.className === 'popInfo') {
+      const popInfo = div;
+      popUpInfo.map((elem) => {
+        elem.classes.map((e) => {
+          if (e === 'popImg') {
+            const div = document.createElement('div');
+            const popImgDiv = div;
+            div.className = e;
+            const popImg = document.createElement('img');
+            popImg.src = elem.image;
+            popImg.alt = elem.alt;
+            popImgDiv.appendChild(popImg);
+            popInfo.appendChild(popImgDiv);
+          }
+        });
```

Comment:
- [ ] Please, as requested by the previous reviewer 🍔, your design should stick as much as possible with the Figma template, and the "See live" and "See source" buttons, should be at the bottom of the popup window, and should have a margin-bottom at the end of the image project. Kindly, fix it 🧇

Your project:

![image](https://user-images.githubusercontent.com/91301423/204093066-3228e3a7-b588-41ae-8941-59d21f7d14d0.png)

Figma design:

![image](https://user-images.githubusercontent.com/91301423/204093122-32894edd-1983-43ab-b663-1f9276bb956f.png)

- [ ] [OPTIONAL] I think it could be better if you center the content of the popup window, it helps to have a similar design to the original. 🥇 

![image](https://user-images.githubusercontent.com/91301423/204093324-a847ed93-860d-4b81-a3e1-fd5df4a29787.png)